examples/resources/hw_documentation/resource.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    dcloud = {
+      version = "0.1"
+      source  = "cisco-open/dcloud"
+    }
+  }
+}
+
+provider "dcloud" {
+  tb_url = "https://tbv3-production.ciscodcloud.com/api"
+}
+
+resource "dcloud_topology" "test_topology" {
+  name        = "My Terraformed Topology"
+  description = "A topology created from terraform"
```

Comment:
Convention on the other resource .tf files is to use these sort of values for the test topology...

```hcl
resource "dcloud_topology" "test_topology" {
  name        = "Documentation Resource Test"
  description = "Testing Topology Documentation Resource Management"
  notes       = "Created via Terraform Test"
  datacenter  = "LON"
}
```

Reply:
Hi Hugh, 

Done. 

Provided the changes in the commit.
Thanks.